### DISCUSSION TITLE
Add a pin button on popout chats for 'Always on top' option

### DISCUSSION THREAD

**Discussion by occluder** [Chatterino/chatterino2#3360] [Ideas] [answered]
It would be really nice to have a button or an option to keep popout chats always on top of other windows., it's a bit annoying right now when you click on something else and the chat goes behind.

example:

![chatterino_8ShAoDHMm5](https://user-images.githubusercontent.com/69414142/142769742-00dcf569-eafb-48d3-8b67-81df244738ec.png)

**Comment by Mm2PL** [comment, marked as answer]
Right click next to tab and check `Topmost window`
